tests/functional/models/appflow-with-more-protocols/2020-08-23/service-2.json

```diff
@@ -0,0 +1,5851 @@
+{
```

Comment:
This is a lot of overhead and makes it difficult to understand what's important in the file. Would it make more sense to do something slimmed down? Either a basic foo service with a single protocol or something like [this](https://github.com/boto/botocore/blob/develop/tests/functional/models/test-discovery-endpoint/2020-08-20/service-2.json)?